gulpfile.js

```diff
@@ -126,34 +126,21 @@ gulp.task( 'compile:themes:esnext', ( callback ) => {
 
 // Building tasks. ------------------------------------------------------------
 
-const ckeditor5DevBundler = require( '@ckeditor/ckeditor5-dev-bundler-rollup' )( config );
-
-gulp.task( 'build', callback => {
-	runSequence(
-		'bundle:generate',
-		[
-			'bundle:minify:js',
-			'bundle:minify:css'
-		],
-		() => ckeditor5DevBundler.showSummaryFromConfig( callback )
-	);
+gulp.task( 'build', () => {
+	const bundler = require( '@ckeditor/ckeditor5-dev-bundler-rollup' );
+
+	return bundler.tasks.build( getBuildOptions() );
 } );
 
-// Helpers. ---------------------------
-
-gulp.task( 'bundle:clean', ckeditor5DevBundler.cleanFromConfig );
-gulp.task( 'bundle:minify:js', ckeditor5DevBundler.minify.jsFromConfig );
-gulp.task( 'bundle:minify:css', ckeditor5DevBundler.minify.cssFromConfig );
-
-// Generates the bundle without minifying it.
-gulp.task( 'bundle:generate',
-	[
-		'bundle:clean',
-		'compile:js:esnext',
```

Comment:
`docs-buider` is still using compiler here, but this part of code can be also moved from the main gulpfile, so here we can expose only `docs`.